packages/icon-library/generate-exports.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs')
```

Comment:
This is way less readable than my bash script...

https://github.com/Royal-Navy/standards-toolkit/pull/379/files#diff-d975a43e2deef42acab44c11f3da4fde

You can run the shell script on windows still??

Not sure I agree with this?